### PR TITLE
Issue 19 - sunrise set times

### PR DESCRIPTION
**Pull request by fluffyemily**
Please make sure to run `pod update` before testing out this branch.

* include EDSunriseSet in Cocoapods
* Update code to calculate Sunrise/set times when location updates
* Display correct sunrise or sunset time in UI
* Allow for simulation of users location to Hawaii Hilton Waikaloa

#22

### CODE REVIEW

**Comment by fluffyemily** [comment]
[Place Carousel Sunset/Sunrise times](https://trello.com/c/vkXysESV/117-place-carousel-sunset-sunrise-times)

File: Prox/Prox/PlaceCarouselViewController.swift [hunk 1]
```diff
@@ -59,6 +62,61 @@ class PlaceCarouselViewController: UIViewController {
...
+
+        sunriseSet.calculateSunriseSunset(today)
+
+        guard var sunrise = sunriseSet.localSunrise(),
+            var sunset = sunriseSet.localSunset(),
+            let calendar = NSCalendar(identifier: NSCalendar.Identifier.gregorian) else {
+                return self.sunriseSetTimesLabel.text = nil
+        }
+
+        sunrise.day = calendar.component(NSCalendar.Unit.day, from: today)
```
Comment (mcomella) [inline Prox/Prox/PlaceCarouselViewController.swift]: We could put this bit of code as a function to reduce code duplication.

File: Prox/Prox/PlaceCarouselViewController.swift [hunk 2]
```diff
@@ -59,6 +62,61 @@ class PlaceCarouselViewController: UIViewController {
...
+        sunrise.month = calendar.component(NSCalendar.Unit.month, from: today)
+        sunrise.year = calendar.component(NSCalendar.Unit.year, from: today)
+
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "h:mm a"
+
+        if let sunriseTime = calendar.date(from: sunrise),
+            sunriseTime.isAfterDate(date: today) {
```
Comment (mcomella) [inline Prox/Prox/PlaceCarouselViewController.swift]: I think you can do `Date(sunriseTime) > today`, here & below.

File: Prox/Prox/PlaceCarouselViewController.swift [hunk 3]
```diff
@@ -59,6 +62,61 @@ class PlaceCarouselViewController: UIViewController {
...
+            let timeAsString = dateFormatter.string(from: sunriseTime)
+            return self.sunriseSetTimesLabel.text = "Sunrise is at \(timeAsString) today"
```
Comment (fluffyemily) [inline Prox/Prox/PlaceCarouselViewController.swift]: raised #37

File: Prox/Prox/PlaceCarouselViewController.swift [hunk 4]
```diff
@@ -206,3 +219,15 @@ extension PlaceCarouselViewController: CLLocationManagerDelegate {
         print("lol-location \(error.localizedDescription)")
     }
 }
+
+
+private extension Date {
```
Comment (fluffyemily) [inline Prox/Prox/PlaceCarouselViewController.swift]: I've moved to use `>`. Even if we kept this, I would prefer to keep the extension private unless we know it is going to be used elsewhere.

**Pull request merged**